yjit/src/options.rs

```diff
@@ -22,6 +22,9 @@ pub struct Options {
     // 1 means always create generic versions
     pub max_versions: usize,
 
+    // The number of registers allocated for stack temps
+    pub temp_regs: usize,
```

Comment:
Suggest renaming to `num_temp_regs` for clarity.

Reply:
fixed so in https://github.com/Shopify/ruby/pull/511/commits/e3db4247ff03b44e8fcb460be47265e69588e336